compliance/mlperf_compliance/mlperf_log.py

```diff
@@ -156,6 +162,13 @@ def ssd_print(key, value=None, stack_offset=1, deferred=False,
                        deferred=deferred, extra_print=extra_print)
 
 
+TRANSFORMER_TAG_SET = set(TRANSFORMER_TAGS)
+def transformer_print(key, value=None, stack_offset=2, deferred=False):
```

Comment:
please change the default stack_offset to 1. Sorry this was set wrongly in my earlier PR.

Reply:
Done. Thanks for catching that.